packages/tools/pie-icons-webc/README.md

```diff
@@ -3,82 +3,88 @@
 
-2. Import a specific icon from its entry point without a reference (recommended)
 ```js
-// Importing a specific icon without keeping a reference from its entry point
-import '@justeattakeaway/pie-icons-webc/IconCalendarFilledLarge';
+// Recommended
+import '@justeattakeaway/pie-icons-webc/dist/IconCalendarFilledLarge.js';
+```
 
-// Rest of code does not directly reference IconCalendarFilledLarge however the web component has been registered in the browser
+The rest of the code does not directly reference an `IconCalendarFilledLarge` object, but the custom element has still been registered in the browser.
+It can now be used inside your HTML template (as long as your JavaScript file is being loaded!).
 
-// In some HTML:
-// <div>
-//   <icon-calendar-filled-large></icon-calendar-filled-large>
-// <div>
+```html
+<div>
+  <icon-calendar-filled-large></icon-calendar-filled-large>
+<div>
 ```
 
-3. Importing all icons at once (not recommended)
+Alternatively, you can import the class, which extends `HTMLElement` (via `LitElement`).
+
 ```js
-// Importing all icons from the library (not recommended)
-import * as icons from '@justeattakeaway/pie-icons-webc';
+// Also recommended, if you need to use the imported object.
+import { IconCalendarFilledLarge } from '@justeattakeaway/pie-icons-webc/dist/IconCalendarFilledLarge.js';
 
 function renderIcon() {
-    // Using a specific icon
-    const iconElement = new icons.IconCalendarFilledLarge();
+    // Using the imported class to create a new element
+    const iconElement = new IconCalendarFilledLarge();
     document.body.appendChild(iconElement);
 }
 ```
 
+#### Importing multiple icons
 
-> [!WARNING]
-> While it is also possible to import individual components from the overall bundle, we don't recommend this because
-> it is likely that the import will be tree-shaken unless you are referencing the imported object in your code.
+The recommended approach for importing multiple icons is to import them one-by-one to keep your application lightweight and performant.
 
 ```js
-// Not recommended -  Webpack v4+ or Rollup should treeshake but be careful
-import { IconAppRestaurant } from '@justeattakeaway/pie-icons-webc';
-// Not recommended
-import { IconAppRestaurant } from '@justeattakeaway/pie-icons-webc';
+import '@justeattakeaway/pie-icons-webc/dist/IconHeart.js';
+import '@justeattakeaway/pie-icons-webc/dist/IconHeartFilled.js';
 ```
 
-#### Lit Components
+```html
+<div>
+  <icon-heart></icon-heart>
+  <icon-heart-filled></icon-heart-filled>
+</div>
+```
+
+Whilst it *is* possible to import all of the icons at once, this is **not recommended** as it will bloat and slow down your application.
+
+Similarly, it is also **not recommended** to import individual icons from the package's main entrypoint, because it is likely that all icons will still be registered as custom elements in the browser.
+
+You may also encounter issues with tree-shaking if you import an object but don't use it.
```

Comment:
Do you know what the issue is with importing an object? Is it worth elaborating further here or is it simply just a case of it stops tree-shaking from working?

Reply:
This is intentionally a little vague because it hasn't been fully explored. At the moment it's more of a warning to avoid some *potential* issues. The consumer's linter will likely complain if the import isn't used anyway, so hopefully there won't be too much of a problem.